srcpkgs/simutron/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'simutron'
+pkgname=simutron
+version=1.0.1
+revision=1
+_rev=SR2
+build_wrksrc=build
+build_style=qmake
+hostmakedepends="subversion qt5-qmake qt5-host-tools pkg-config"
+makedepends="simavr-devel qt5-devel"
+depends="libelf"
+short_desc="AVR simulator IDE"
+maintainer="Artur Sinila <opensource@logarithmus.dev>"
+license="GPL-3.0-only"
+homepage="https://sourceforge.net/projects/simutron/"
+nostrip_files="LCD20x4Test.elf"
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-host-tools"
+fi
```

Comment:
remove, as this already is in hostmakedepends?

Reply:
I rather removed it from `hostmakedepends`

Reply:
1. you didn't yet
2. keep it in `hostmakedepends` and remove that conditional

Reply:
Just did it the way you wrote.